package.json

```diff
@@ -3,18 +3,40 @@
   "version": "0.0.1",
   "author": "Mobify",
   "dependencies": {
-    "astro-sdk": "~0.17.0",
-    "bluebird": "~2.9.24"
+    "babel-runtime": "^6.20.0",
+    "bluebird": "~2.9.24",
+    "mobify-progressive-app-sdk": "~0.18.0",
+    "webpack": "^2.1.0-beta.27"
```

Comment:
YES. Oh man, I _think_ that this change fixed the back button issue I was experiencing last week (back button wouldn't show up in the header on iOS during navigation). OR, it's a race condition and I'm on the right side of the race right now. But, I think this fixed it so we'll go with that for now.